src/less/toggle-button/toggle-button.less

```diff
@@ -0,0 +1,123 @@
+@import "../variables/variables.less";
+
+:root {
+    --toggle-button-height-min: 72px;
+    --toggle-button-width-max: 342px;
+    --toggle-button-width-min: 224px;
+    --toggle-button-minimal-width-min: 72px;
+    --toggle-button-minimal-width-max: 168px;
+    --toggle-button-gallery-width-min: 168px;
+    --toggle-button-gallery-width-max: 342px;
+}
+
+.toggle-button {
+    background-color: var(--color-background-primary);
+    border: 1px solid var(--color-stroke-default);
+    border-radius: @spacing-100;
+    box-shadow: 0 0 0 1px transparent;
+    color: var(--color-foreground-primary);
+    display: inline-flex;
+    font-size: @font-size-14;
+    margin-right: @spacing-100;
+    margin-top: auto;
+    max-width: var(--toggle-button-width-max);
+    min-height: var(--toggle-button-height-min);
+    min-width: var(--toggle-button-width-min);
+    padding: 0 @spacing-200;
+    vertical-align: top;
+
+    &:focus-visible {
+        outline-offset: 1px;
+        outline-style: outset;
+        outline-width: 2px;
+    }
+}
+
+.toggle-button[aria-pressed="true"] {
+    background-color: var(--color-background-secondary);
+    border: 1px solid var(--color-stroke-strong);
+    box-shadow: 0 0 0 1px var(--color-stroke-strong);
+    font-weight: bold;
+}
+
+.toggle-button__content {
+    align-self: center;
+    display: flex;
+    flex-direction: column;
+    text-align: left;
+}
+
+.toggle-button__icon {
+    margin-bottom: @spacing-200;
+    margin-right: @spacing-200;
+    margin-top: @spacing-200;
+}
+
+.toggle-button__image {
```

Comment:
I'm not sure if this is the right place to fix this, but it looks like toggle buttons with images which are less than the full width of the container push the left side of the text towards the left side of the image.

It looks like this:
<img width="383" alt="image" src="https://github.com/eBay/skin/assets/26027232/29d5ad3c-2c8b-4365-84c1-05caecd29ba1">

When it should look like this:
<img width="219" alt="image" src="https://github.com/eBay/skin/assets/26027232/2fc22a98-f1af-44ed-ab2c-13a833eeb0e8">

Reply:
It's definitely the right place to fix the text alignment on gallery view; should be left-aligned. Good catch! I'll fix it.

I'm not sure what you meant with this though:

> buttons with images which are less than the full width of the container push the left side of the text towards the left side of the image

To clarify (if I'm reading the context correctly), the text and image would never be expected to be aligned together inside buttons because the text will be left aligned and the image is center aligned AND we can't enforce/anticipate what kind of image will be used inside.